plugins/modules/vmadm.py

```diff
@@ -104,6 +104,11 @@
     description:
       - Enables the firewall, allowing fwadm(1M) rules to be applied.
     type: bool
+  flexible_disk_size:
+    required: false
+    description:
+      - This sets an upper bound for the amount of space that a bhyve instance may use for its disks and snapshots of those disks (in MiBs)
```

Comment:
```suggestion
      - This sets an upper bound for the amount of space that a bhyve instance may use for its disks and snapshots of those disks (in MiBs).
```

Reply:
What is "a bhvye instance"? I reckon that term should have some special formatting.

Reply:
Do you have a suggestion for the formatting? I am unfamiliar with what Ansible would need in this case. bhyve is a hypervisor developed by FreeBSD, and is one of the hypervisors SmartOS uses. This is out of the vmadm man page
https://smartos.org/man/8/vmadm

Reply:
According to https://en.wikipedia.org/wiki/Bhyve bhyve is the product name, so just writing it like that should be fine. I guess FreeBSD users/admins are more familiar with that term (I had to search for it as well, but then I don't use FreeBSD :) ).